### ISSUE TITLE
Wikisource शब्दकल्पद्रुमः Edition

### ISSUE THREAD

**Issue by gasyoun** [sanskrit-lexicon/SKD#6]
@drdhaval2785 so there is SKD in Bengal and Devanagari fonts, at least 2 different paper editions. I wonder  if https://sa.wikisource.org/wiki/%E0%A4%B6%E0%A4%AC%E0%A5%8D%E0%A4%A6%E0%A4%95%E0%A4%B2%E0%A5%8D%E0%A4%AA%E0%A4%A6%E0%A5%8D%E0%A4%B0%E0%A5%81%E0%A4%AE%E0%A4%83/%E0%A4%B5%E0%A4%BF%E0%A4%A7 is based on the Tirupati edition, @funderburkjim .

**Comment by funderburkjim** [comment]
A.F.A.I.K,  VCP is the one with a Tirupati edition, not SKD.

I see a mention of 'Stardict' on the [home page](https://sa.wikisource.org/wiki/%E0%A4%B6%E0%A4%AC%E0%A5%8D%E0%A4%A6%E0%A4%95%E0%A4%B2%E0%A5%8D%E0%A4%AA%E0%A4%A6%E0%A5%8D%E0%A4%B0%E0%A5%81%E0%A4%AE%E0%A4%83).  

The main language of the wiki seems to be Hindi.  Maybe Dhaval could interpret it and determine the
source of the digitization.  If the digitization is not that of Cologne,  i.e., if there is an independent digitization of skd,  then it might prove useful as comparison to the Cologne digitization.

Based solely on a comparison of the first few lines of headword [a](https://sa.wikisource.org/wiki/%E0%A4%B6%E0%A4%AC%E0%A5%8D%E0%A4%A6%E0%A4%95%E0%A4%B2%E0%A5%8D%E0%A4%AA%E0%A4%A6%E0%A5%8D%E0%A4%B0%E0%A5%81%E0%A4%AE%E0%A4%83/%E0%A4%85) to the Cologne digitization, the wikisource edition appears the same as Cologne.

**Comment by gasyoun** [comment]
> Based solely on a comparison of the first few lines of headword a to the Cologne digitization, the wikisource edition appears the same as Cologne.

Oh, might be. Like the sitewide representation, miss it on Cologne.

**Comment by funderburkjim** [comment]
> Like the sitewide representation, 

I agree.  Its a nice display.  The first page provides a sort of jump table.   
I also like the wiki idea, as it allows the possibility of having a local copy of a dictionary, where the 
user could (via the wiki process) made personal annotations.

**Comment by gasyoun** [comment]
> I agree. Its a nice display. 

Actually there are advanced users who open Page view every time, just to see what's around. They use Cologne only as a way to open scanned images quicker because there is no way to open the digital copy of a page.
